schema/schema.go

```diff
@@ -27,6 +27,7 @@ var searchDataImportEvent = `{
     {"name": "published", "type": "boolean", "default": false},
     {"name": "language", "type": "string", "default": ""},
     {"name": "survey",   "type": "string", "default": ""},
+    {"name": "canonical_topic",   "type": "string", "default": ""},
```

Comment:
Aren't there extra whitespaces here?
```suggestion
    {"name": "canonical_topic", "type": "string", "default": ""},
```

It seems to have been copied from the line above, survey, which also has extra spaces